bundles/net.tourbook/src/net/tourbook/messages_fr.properties

```diff
@@ -729,14 +731,14 @@ Dlg_AdjustAltitude_Radio_keep_start_altitude_tooltip  = Garder l'altitude du d\u
 Dlg_TourMarker_Button_HideAllMarker              = M&asquer tout
 Dlg_TourMarker_Button_HideAllMarker_Tooltip      = Masquer tous les marqueurs
 Dlg_TourMarker_Button_PasteFromClipboard_Tooltip = Ins\u00E9rer le texte depuis le presse-papier et le convertir afin de le rendre plus lisible.\n\
-                                                   \n\
-                                                   Exemple: \n\
-                                                   \n\
-                                                   Presse Papier : \n\
-                                                   http://fr.wikipedia.org/wiki/L%27Alpe_d%27Huez\n\
-                                                   \n\
-                                                   Texte corrig\u00E9 : \n\
-                                                   http://fr.wikipedia.org/wiki/L'Alpe_d'Huez
+\n\

```

Comment:
@wolfgang-ch Here is an example of what happens when I just save my french translations using with the Messages Editor. Do you know why this happens ?

Reply:
In my comment from Aug 23 https://github.com/wolfgang-ch/mytourbook/pull/95#pullrequestreview-279078143 I pointed out that long lines are formatted differently but the conent which is displayed to the user is the same

Reply:
Ah true. I suppose if it's displayed to the user the same it's good. 
The only problem is that if i modify an english translation file and then you modify after me, then if I modify again, in the end it makes a lot of "noise".

Reply:
>in the end it makes a lot of "noise".

When you _Ignore White Space_, there is no noise, the yellow area is not marked as modified and this is the only difference I know, between the two messages editors

I modify other language files mainly, when adding new images that the translators do not have to "translate" it.

![compare-noise](https://user-images.githubusercontent.com/1283445/67155142-a3ceb900-f309-11e9-8bf8-433cf8f1d291.png)